src/EmotionRecognitionTask.ts

```diff
@@ -215,11 +224,12 @@ export default async function emotionRecognitionTask(onFinish?: (data: EmotionRe
         const audioContent = document.getElementById('audioContent');
 
         let start_time = 0;
```

Comment:
use camelCase for everything, unless there is some reason not to (i.e., JsPsych options)